### PR TITLE
Patch confirmed date when status is set to Active 

### PR DESCRIPTION
**Pull request by dzldebby**
# Patch confirmed date when status is set to Active on Matched table 

Currently, confirmed date is not updated when status of match is set to Active. 

## Description

This change updates the confirmed date once the status of the Match is set to `Active` on the Matched table. 

## Test Steps

1. Create a new Student
2. Create a new Teacher
3. Match the Student and the Teacher
4. Go to Matched Table 
5. Select 'Active' as the Confirmation field
6. The `Confirmed On` date field should be updated to the date that step 5 was done

### CODE REVIEW

File: backend/src/helpers/matches.js [hunk 1]
```diff
@@ -19,7 +19,12 @@ exports.patchMatchStatus = async function (MatchID, MatchStatus) {
   try {
     let query = await db.Match.query()
       .findById(MatchID)
-      .patch({ MatchStatus: MatchStatus });
+      .patch(
+        {
+           MatchStatus: MatchStatus,
+           ConfirmedDate: new Date()
```
Comment (thchia) [inline backend/src/helpers/matches.js:25]: Do we update the `ConfirmedDate` no matter what the updated `MatchStatus` is? Or only in certain cases (not been following this feature so not sure how it works).

**Review by thchia — APPROVED** [review]
One comment that may or may not need a change. If not, take this as an approval.

**Review by nondescryptid — APPROVED** [review]
tested, LGTM!

File: backend/src/helpers/matches.js (hunk 1 above)
Comment (dzldebby) [reply backend/src/helpers/matches.js:25]: As discussed, currently the `MatchStatus` can only be toggled from "Pending" to "Active". Marking as resolved.

**Pull request merged**